[O11y][ActiveMQ] Update README (#8926)
elastic/integrations: packages/activemq/_dev/build/docs/README.md

## Compatibility

This integration has been tested against ActiveMQ 5.17.1 (independent from the operating system).

## Prerequisites

You need Elasticsearch to store and search your data and Kibana to visualize and manage it.
You can use our hosted Elasticsearch Service on Elastic Cloud, which is recommended or self-manage the Elastic Stack on your hardware.

## Setup

For step-by-step instructions on how to set up an integration, see the [Getting Started](https://www.elastic.co/guide/en/welcome-to-elastic/current/getting-started-observability.html) guide.

## Supported log formats for Audit and ActiveMQ logs

Here are the supported log format for the Audit logs and ActiveMQ logs in the ActiveMQ instance,

### Audit

```
%-5p | %m | %t%n
```

Here is the breakdown of the pattern:

- %-5p: This part represents the log level left-aligned with a width of 5 characters. The - signifies left alignment.

- %m: This part represents the log message.

- %t%n: This part represents the thread name (%t) followed by a newline (%n).

> ### ActiveMQ

Reviewer: We need to change this title to either Logs or ActiveMQ Logs. Please change this based on the naming conventions followed below for Logs datastream.

- %m: This part represents the log message.

- %t%n: This part represents the thread name (%t) followed by a newline (%n).

### ActiveMQ

```
%d | %-5p | %m | %c | %t%n%throwable{full}
```

Here is the breakdown of the pattern:
- %d: This part represents the date and time of the log event in the ISO8601 format.

- %-5p: This part represents the log level left-aligned with a width of 5 characters. The - signifies left alignment.

- %m: This part represents the log message.

- %c: This part represents the logger category (class name).

- %t%n: This part represents the thread name (%t) followed by a newline (%n).

- %throwable{full}: This part represents the full stack trace if an exception is attached to the log entry.

## Validation

After the integration is successfully configured, clicking on the Assets tab of the ActiveMQ Integration should display a list of available dashboards. Click on the dashboard available for your configured data stream. It should be populated with the required data.

## Logs

### ActiveMQ Logs

These logs are System logs of ActiveMQ.

{{event "log"}}

{{fields "log"}}

### Audit Logs

> For many secured environments, there's a requirement to log every user management action. For that ActiveMQ implements audit logging, which means that every management action made through JMX or Web Console management interface will be logged and available for later inspection.

Reviewer 2: ```suggestion
In secured environments, it is required to log every user management action. ActiveMQ implements audit logging, which means that every management action made through JMX or Web Console management interface is logged and available for later inspection.
```

### Broker Metrics

ActiveMQ brokers serve as implementations of the Java Messaging Service (JMS), a Java specification facilitating the seamless exchange of data between applications. Metrics provide insights into statistics such as enqueued and dequeued messages, as well as details on consumers, producers, and memory usage (broker, store, temp).

{{event "broker"}}

{{fields "broker"}}

### Queue Metrics

> Queues are FIFO (first-in, first-out) pipelines of messages produced and consumed by brokers and clients. Producers create messages and push them onto these queues. Then those messages are polled and collected by consumer applications, one message at a time. Metrics show statistics of exchanged messages, consumers, producers and memory usage.

Reviewer 2: ```suggestion
Queues are FIFO (first-in, first-out) pipelines of messages produced and consumed by brokers and clients. Producers create messages and push them onto these queues. Then, those messages are polled and collected by consumer applications, one message at a time. Metrics show statistics of exchanged messages, consumers, producers and memory usage.
```
